src/Controls/tests/DeviceTests/Memory/MemoryTests.cs

```diff
@@ -132,7 +132,6 @@ await CreateHandlerAndAddToWindow(new Window(navPage), async () =>
 	[InlineData(typeof(CarouselView))]
 	[InlineData(typeof(ContentView))]
 	[InlineData(typeof(CheckBox))]
-	[InlineData(typeof(DatePicker))]
```

Comment:
Instead of deleting this can you add a comment here

// This test was moved to MemoryTests.cs inside Appium